### DISCUSSION TITLE
Run DotSpatial.Positioning under Linux

### DISCUSSION THREAD

**Discussion by lvv83** [DotSpatial/DotSpatial#1543] [Q&A]
Hello.
Is there any chances to run code from DotSpatial.Positioning library under Linux?
I see, that there are only windows libraries on [NuGet](https://www.nuget.org/packages/DotSpatial.Positioning#supportedframeworks-body-tab).

Thanks!

**Comment by sindizzy** [comment]
Run Wine on linux

**Reply by lvv83** [reply]
Are there any plans for Net.Core port?
We use it on server (web application)

**Reply by sindizzy** [reply]
It's already on .net core. Take a look at the main page on GitHub.
